### ISSUE TITLE
On device activity is not working again

### ISSUE THREAD

**Issue by obaldius** [cakebake/node-red-contrib-alexa-remote-cakebaked#30]
On device activity node has stopped working in the last hours. Is anybody experiencing this issue as well?

**Comment by notownblues** [comment]
Yes can confirm On device activity has stopped working for me too.

**Comment by obaldius** [comment]
thank God it's not just me then :)

**Comment by Thymos2k** [comment]
Can confirm it, too

**Comment by bbindreiter** [comment]
Yes it's gone for me too. Let's see if it's just an outage or Amazon changed something. If they have we need @Apollon77

**Comment by Apollon77** [comment]
Grmpf ... I check what they did or if it is an outage ...

tracking on fix will be in https://github.com/Apollon77/ioBroker.alexa2/issues/719

**Comment by Apollon77** [comment]
Try again ... started magicslly working again 20 mins ago ... so seems to be an amazon outage was reason

**Comment by obaldius** [comment]
> Try again ... started magicslly working again 20 mins ago ... so seems to be an amazon outage was reason

Yep, it's back to normal :D

**Comment by obaldius** [comment]
Has somebody experienced some unstability with the On device activity node ever since the outage?

I feel it fails more often than before.. maybe one out of 10 times the On device activity doesn't catch the event.

**Comment by ammawel** [comment]
Yes, I can confirm the issues.
For me, the "Alexa-Smarthome" node with the query function and the voice output from Alexa also fail more frequently.
So far I have done an init every morning via an inject node, but this is not enough anymore.

The whole init process seems to be bugy, it looks like 586837r didn't fully use Apollon77's init process from the "alexa-remote" module, but wrote his own functions for the init. These are of course old and have not been adapted with the updates of Apollon77's dependencies. 

How do you control the init process? With an inject node or via the account definition? What settings do you have?

Greetings
Achim

**Comment by obaldius** [comment]
> How do you control the init process? With an inject node or via the account definition? What settings do you have?

I'm not sure if this is what you mean by I've been using this flow as a counter measuer to the account initialization problems:

https://pastebin.com/VG6y29XB

Still I'm experiencing more issues tha before. The smarthome feature under the routine node has also failed on me. Not a lot, but definetely more than before the outage.

**Comment by FenvyWorks** [comment]
Did anyone find a fix for this? it stopped working 3 days ago for me..

**Comment by obaldius** [comment]
I'm still experiencing unstability issues with almos every node. It's not that often, maybe a 10% of the times, but every now and then something doesn't get run even though the node itself states "success"
